src/current/v24.3/backup.md

```diff
@@ -32,7 +32,7 @@ To view the contents of an backup created with the `BACKUP` statement, use [`SHO
 
 ## Considerations
 
-- Core users can only take [full backups]({% link {{ page.version.version }}/take-full-and-incremental-backups.md %}#full-backups). To use the other backup features, you need an [Enterprise license]({% link {{ page.version.version }}/enterprise-licensing.md %}). You can also use [CockroachDB {{ site.data.products.advanced }}](https://cockroachlabs.cloud/signup?referralId=docs-crdb-backup), which runs [full backups daily and incremental backups hourly]({% link cockroachcloud/managed-backups.md %}).
+If you use [CockroachDB {{ site.data.products.advanced }}](https://cockroachlabs.cloud/signup?referralId=docs-crdb-backup), it runs [full backups daily and incremental backups hourly]({% link cockroachcloud/managed-backups.md %}).
```

Comment:
I think we should just remove this bullet point.

Reply:
Removed the whole thing! thanks